glossary.md

```diff
@@ -1,22 +1,149 @@
+* **CLDR**: [Common Locale Data Repository][] - A repository of locale data used in software engineering.
+* **CLI**: [Command Line Interface][] - A way to interact with a computer program using text commands.
+* **Code cache**: Chunk of bytes storing compiled JS code and its metadata.
+* **CVE**: [Common Vulnerabilities and Exposures][] - Database maintaining reported security vulnerabilities.
+* **Deps**: Dependencies - Upstream projects that this project depends on.
+* **DOM**: [Document Object Model][] - A programming interface for web documents. It represents the structure of a
+  document as a tree of objects, allowing programmers to manipulate the content and structure of a web page dynamically.
+* **ECMA**: [European Computer Manufacturers Association][] - Governing body over JavaScript's specification.
+* **ECMA-262**: [**ECMA**'s standard for JavaScript][].
+* **ECMAScript**: JavaScript, as described in **ECMA-262**.
+* **EOF**: [End-of-File][] - Indicates the end of a file or stream.
+* **EOL**: [End-of-Life][] (when used within project documents), [End-of-Line][] (when used within a program),
+  End-of-Life is usually how this term is used.
+* **ESM**: [ECMAScript Module][] - Standard for JavaScript modules.
+* **ETW**: [Event Tracing for Windows][] - Provides a way to trace events in Windows systems.
+* **FFDC**: First Failure Data Capture - Logs, traces, and dumps produced by default on program error.
+* **FIPS**: [Federal Information Processing Standards][] - Set of standards for use in computer systems by non-military
+  government agencies and government contractors.
+* **FS**: File System.
+* **Godbolt**: [Compiler Explorer][] - Tool for running compilers interactively from a web browser.
+* **HTTP**: [Hypertext Transfer Protocol][] - An application protocol for distributed, collaborative, hypermedia
+  information systems. It is the foundation of data communication on the World Wide Web.
+* **ICU**: [International Components for Unicode][] - Library providing support for Unicode.
+* **IDE**: [Integrated Development Environment][] - A software application that provides comprehensive facilities to
+  computer programmers for software development.
+* **IETF**: [Internet Engineering Task Force][] - An international community responsible for developing and promoting
+  Internet standards.
+* **IIRC**: If I Recall Correctly.
+* **IIUC**: If I Understand Correctly.
+* **IMHO**: In My Humble/Honest Opinion.
+* **IMO**: In My Opinion.
+* **IPC**: [Inter-Process Communication][] - Mechanism allowing processes to communicate with each other.
+* **JIT**: [Just In Time][] - Method of executing computer code during runtime.
+* **JS**: [JavaScript][] - A high-level, interpreted programming language that is primarily used to create dynamic and
+  interactive content on web pages. It is commonly used for client-side scripting in web development.
```

Comment:
I think describing JS as "primarily used to create dynamic and interactive content on web pages" would not really fit what folks use JS in the context of Node.js
```suggestion
* **JS**: [JavaScript][] - A high-level, interpreted programming language that conforms to the **ECMAScript** specification.
```